integration/insert_command_test.go

```diff
@@ -132,3 +132,41 @@ func TestInsertCommandErrors(tt *testing.T) {
 		})
 	}
 }
+
+func TestInsertIDDifferentTypes(t *testing.T) {
```

Comment:
```suggestion
func TestInsertIDDuplicateNumberTypes(t *testing.T) {
```
Something a bit more specific to what it's testing would be better. By no means, it's not a blocker :hugs:

Reply:
Let's keep as-is for now